core/deviceadaptor.cpp

```diff
@@ -120,10 +120,12 @@ RingBufferBase* DeviceAdaptor::findBuffer(const QString& name) const
 bool DeviceAdaptor::setStandbyOverride(bool override)
 {
     standbyOverride_ = override;
-    if (override) {
-        resume();
-    } else {
-        standby();
+    if (screenBlanked_) {
+        if (override) {
+            resume();
+        } else {
+            standby();
+        }
```

Comment:
I am only adding this back.
The reason I removed it before is that I thought I was seeing some delay in blanking, but was really only caused by the debug output I am also removing here, which led me to believe standby was being called when it really wasn't.